Assets/Scripts/Models/World.cs

```diff
@@ -342,6 +342,44 @@ public Utility PlaceUtility(string objectType, Tile tile, bool doRoomFloodFill =
         return PlaceUtility(util, tile, doRoomFloodFill);
     }
 
+    /// <summary>
+    /// Reserves the furniture's work spot, preventing it from being built on.
+    /// </summary>
+    /// <param name="furn">The furniture whose workspot will be reserved.</param>
+    /// <param name="tile">The tile on which the furniture is located, for furnitures which don't have a tile, such as prototypes.</param>
+    public void ReserveTileAsWorkSpot(Furniture furn, Tile tile = null)
```

Comment:
Sorry. Was going to do it as 1 line and went with 2 since it was cleaner. I am not sure as to which way is cleaner. Most things are in Tile, so I figure it might work better there. But both ways are correct.